.env.demo

```
@@ -0,0 +1,28 @@
+# Database (Railway will provide)
+DATABASE_URL=${DATABASE_URL}
+REDIS_URL=${REDIS_URL}
+
+# Application Settings
+APP_NAME="Austrian Hospitality Leads - Andy's Demo"
+APP_URL=https://andy-demo.austrialeads.com
+API_URL=https://andy-demo.austrialeads.com/api
+
+# Session Configuration
+SESSION_SECRET=demo-session-secret-change-in-production
+SESSION_DURATION=86400000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SESSION_DURATION key should go before the SESSION_SECRET key

(UnorderedKey)